cmd/tracee-rules/main.go

```diff
@@ -222,6 +228,11 @@ func main() {
 				Usage: "listening address of the metrics endpoint server",
 				Value: ":4466",
 			},
+			&cli.BoolFlag{
```

Comment:
Maybe just add a shorter alias for the flag (like --ahc or something) for usability? Because it's a bit verbose rn.

Reply:
Yep, this is very big.. maybe we should address this now ?